src/styles.scss

```diff
@@ -274,3 +274,11 @@ $standard-icon-size: 1.5em;
 .margin-bottom-small {
   margin-bottom: $standard-margin-small;
 }
+
+/**
+ * Flex
+ */
+
+.flex-grow {
```

Comment:
isn't this something FlexLayoutModule can provide?
@Schottkyc137, wanna keep an eye on this while refactoring and cleaning up the styles?

Reply:
I actually do believe that this is a better way of doing things (I am using a similar approach while refactoring). I actually think that we should get rid of FxFlex altogether. We will probably have a discussion about this topic when I'm ready for review but the basic reasoning is that CSS flex classes are in most cases (all that I have encountered in the app) just as potent and – at least from my point of view – more clear than fxFlex. The latter works using "magic" by using styles that are added dynamically via scripts. This is almost never needed.